backend/director/agents/web_search_agent.py

```
@@ -0,0 +1,172 @@
+import logging
+import os
+from dotenv import load_dotenv
+
+from director.agents.base import BaseAgent, AgentResponse, AgentStatus
+from director.core.session import (
+    Session,
+    TextContent,
```

Ruff (0.8.2)

`director.core.session.TextContent` imported but unused

Remove unused import: `director.core.session.TextContent`

(F401)